docs/en/userguide/02_Administrator_manual/02_Resetting_accounts.md

```diff
@@ -0,0 +1,21 @@
+title: Resetting accounts
+summary: How to reset a user's account if they lose access to it
+
+# Resetting accounts
+
+As an administrator, you can assist other users that have been locked out of
+their accounts by sending them an Account Reset request via the CMS.
+
+<img src="../_images/02-02-1-account-reset.png" width="1024" />
+
+1. Verify that the person requesting the reset owns the account.
+2. Navigate to the **Security** section, and select the member requesting an
+   account reset.
+3. Locate the **Multi-factor Authentication settings (MFA)** field, and if
+   prompted, re-enter your password to unlock it. Press
+   **Send account reset email**, and confirm the action in the dialog.
+
+<img src="../_images/02-02-2-account-reset-confirmation.png" width="1024" />
```

Comment:
Minor comment. The style of modals is different in SS4.4 - I notice this is 4.3 or something

Reply:
Good spotting - my local is indeed 4.3-based. I think this is the only screenshot that this affects, so it's probably not worth upgrading my environment just to retake it at this stage.